### ISSUE TITLE
DEP: Create a package for libecpg on conda-forge

### ISSUE THREAD

**Issue by xmnlab** [thegraphnetwork/EpiGraphHub#174]
It would be nice to have libecpg on conda-forge.
currently there is something here: https://anaconda.org/biobuilds/libecpg
maybe a good approach for that would be:

- download the package from https://anaconda.org/biobuilds/libecpg
- extract the file and check the recipe
- replicate that and open a PR to conda-forge/staged-recipe

ref: #171

**Comment by fccoelho** [comment]
Why not just use the deb package?

**Comment by xmnlab** [comment]
yep for now it is a deb package, but if we would like to install it locally for testing, for example, it would be better to have it from conda-forge .. so it would work easily for any machine architecture, it is not a priority .. 
I added it with "help wanted" label, so maybe if someone is learning packing and wants to practice, it would be a nice one :) 

also it is a nice way to contribute back to the community :)
